adding holiday outages to get help page (#507)
CHTC/chtc-website-source: uw-research-computing/get-help.md

## Help Via Email

> ### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Expect delayed replies to email during this time. 

Reviewer: ```suggestion
### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Office hours will be cancelled during this period, and users should expect delayed replies to emails.
```

> ### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Office hours will be cancelled and users should expect delayed replies to emails during this period.

Reviewer: ```suggestion
### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Users should expect delayed replies to emails during this period.
```

> ### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Office hours will be cancelled during this period, and users should expect delayed replies to emails.

Reviewer: ```suggestion
### Facilitation staff will be out of the office Monday, December 26 through Monday, January 2. Office hours will be cancelled and users should expect delayed replies to emails during this period.
```
